contiBAIT v1.1.7

rareaquaticbadger opened the pull request:
# 

Function changes:
- strandSeqFreqTable now takes in a chimera file, the product of the second element of LocateMisroients.
- OrientationFrame has been depreciated. ReorientAndMergeLGs now generates a ChrTable with the strand information rather than the OrientationFrame data.frame
- ideogramPlot has been changed to accept the ChrTable orientation element
- writeBed has been changed to accept the ChrTable orientation element
- preProcessStrandTable now returns a ChrTable of AWC regions, rather than a vector

Minor changes:
- plotContigOrder tidied up
- plotLGDistances tidied up
- barplotLinkageGroupCalls tidied up

rareaquaticbadger committed:
contiBAIT v1.1.7
==============
Function changes:
- strandSeqFreqTable now takes in a chimera file, the product of the second element of LocateMisroients.
- OrientationFrame has been depreciated. ReorientAndMergeLGs now generates a ChrTable with the strand information rather than the OrientationFrame data.frame
- ideogramPlot has been changed to accept the ChrTable orientation element
- writeBed has been changed to accept the ChrTable orientation element
- preProcessStrandTable now returns a ChrTable of AWC regions, rather than a vector

Minor changes:
- plotContigOrder tidied up
- plotLGDistances tidied up
- barplotLinkageGroupCalls tidied up

rareaquaticbadger committed:
contiBAIT v1.1.7
==============
Function changes:
- strandSeqFreqTable now takes in a chimera file, the product of the second element of LocateMisroients.
- OrientationFrame has been depreciated. ReorientAndMergeLGs now generates a ChrTable with the strand information rather than the OrientationFrame data.frame
- ideogramPlot has been changed to accept the ChrTable orientation element
- writeBed has been changed to accept the ChrTable orientation element
- preProcessStrandTable now returns a ChrTable of AWC regions, rather than a vector

Minor changes:
- plotContigOrder tidied up
- plotLGDistances tidied up
- barplotLinkageGroupCalls tidied up

oneillkza commented:
Look like there are one or two minor errors in the documentation / new function definitions:

```
* checking Rd \usage sections ... WARNING
Undocumented arguments in documentation object 'strandSeqFreqTable'
  ‘chimeras’
Duplicated \argument entries in documentation object 'strandSeqFreqTable':
  ‘misorientations’
```

rareaquaticbadger commented:
Ah, sorry, I roxygenized, and it built fine, but I forgot to commit the doc file. should be fixed now...

rareaquaticbadger committed:
Merge branch 'master' of https://github.com/rareaquaticbadger/ContiBAIT

Conflicts:
	man/strandSeqFreqTable.Rd

rareaquaticbadger committed:
Merge branch 'master' of https://github.com/rareaquaticbadger/ContiBAIT

Conflicts:
	man/strandSeqFreqTable.Rd

rareaquaticbadger committed:
a

Merge branch 'master' of https://github.com/rareaquaticbadger/ContiBAIT

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

oneillkza commented:
Haha no worries! And thanks!
